src/commands/Info/mutual.js

```diff
@@ -8,21 +8,22 @@ exports.run = async (bot, msg, args) => {
     }
 
     const query = args.join(' ').toLowerCase();
-    const guild = bot.guilds.find(guild => guild.name.toLowerCase() === query);
-
-    if (!guild) {
-        throw 'That guild could not be found!';
-    }
-
+	// Try to parse by Server Name fist or Server ID
```

Comment:
Here and on lines 14-16, the indentation is incorrect. Please fix this, and then I'll accept your PR :)